Stratis.Bitcoin.Features.LightWallet/LightWalletFeePolicy.cs

```diff
@@ -29,29 +30,63 @@ public LightWalletFeePolicy(IAsyncLoopFactory asyncLoopFactory, INodeLifetime no
             this.highTxFeePerKb = null;
```

Comment:
I think these are null by default, no need to initialize them here (although the compiler might disagree).